File: ord/src/main/java/dag/ord/util/UrlReader.java
package dag.ord.util;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.net.URL;

public class UrlReader {

    public static String read(String urlString) throws IOException {
        StringBuilder sb = new StringBuilder();

        URL url = new URL(urlString);
        BufferedReader in = new BufferedReader(
                new InputStreamReader(url.openStream()));

        String inputLine;
        while ((inputLine = in.readLine()) != null) {
            sb.append(inputLine);
        }
        in.close();

        return sb.toString();
    }
}

File: ord/src/main/java/dag/ord/util/UrlReader.kt
package dag.ord.util

import okhttp3.OkHttpClient
import okhttp3.Request
import java.io.*
import java.net.URL

object UrlReader {
    private val httpClient = OkHttpClient()

    fun read(url: String) : String? {
        val request = Request.Builder()
            .url(url)
            .build()

        return httpClient.newCall(request).execute().body?.string()
    }
}
